web/js/utils-0.0.8.js

```diff
@@ -1634,6 +1674,7 @@ function pageReadyList() {
     }
     $('#navigate').click(function () {
         $.navigateWindow.toggle()
+        return false;
```

Comment:
read as "result is that it wasn't clicked on this element" - this doesn't add the empty # in the url